10_date_timestamps_and_lies.md

```diff
@@ -106,8 +106,6 @@ val df = sqlContext.read.format("jdbc")
   .load
 ```
 
-But ***why doesn't casting work ?***
```

Comment:
I don't see any place where you actually try cast to string but my main point is that we could put it as a general rule: type casting is performed on the Spark side, not on the source side. I think it matters only (mostly?) for JDBC sources.